src/components/User/SignIn.js

```diff
@@ -3,8 +3,18 @@ import Navbar from "./Common/Navbar";
 import Switchcomp from "./Common/Switchcomp/Switchcomp";
 import Input from "./Common/Input";
 import { navigate } from "hookrouter";
+import GoogleLogin from "react-google-login";
+
+//for testing  . Need to be put in .env file
+const clientID =
+    "96926798815-lcth5an8pa6poe6ksqlnj4ms45ilis8p.apps.googleusercontent.com";
```

Comment:
use `process.env. REACT_APP_GOOGLE_CLIENT_ID` here and remove this hard coded valuev

Reply:
Add  it to .env.example as empty value

Reply:
@gautamkrishnar lets use this endpoint https://api.hellomunnar.in/google instead. let fe call this endpoint

Reply:
Yes @tomahawk-pilot, we can use that value after deployment. Lets move this to .env instead of hardcoding anything. The env.example should only have empty value for the corresponding env.

Reply:
@tomahawk-pilot  Please sync up with @psjishnu and help him with the API.